config/settings.yml

```diff
@@ -1036,7 +1036,7 @@
       - vmtx
       - vswp
       - "%redo%"
-  :max_parallel_scans_per_host: 1
+  :max_parallel_scans_per_ems: 5
```

Comment:
@Fryguy does there need to be a migration for this settings change?

Reply:
Yes.